packages/http-client-python/package.json

```
@@ -51,41 +51,41 @@
     "emitter"
   ],
   "peerDependencies": {
-    "@azure-tools/typespec-azure-core": ">=0.46.0 <1.0.0",
-    "@azure-tools/typespec-azure-resource-manager": ">=0.46.0 <1.0.0",
-    "@azure-tools/typespec-autorest": ">=0.46.0 <1.0.0",
-    "@azure-tools/typespec-client-generator-core": ">=0.46.2 <1.0.0",
-    "@azure-tools/typespec-azure-rulesets": ">=0.46.0 <3.0.0",
-    "@typespec/compiler": ">=0.60.0 <1.0.0",
-    "@typespec/http": ">=0.60.0 <1.0.0",
-    "@typespec/rest": ">=0.60.0 <1.0.0",
-    "@typespec/versioning": ">=0.60.0 <1.0.0",
-    "@typespec/openapi": ">=0.60.0 <1.0.0"
+    "@azure-tools/typespec-azure-core": ">=0.47.0 <1.0.0",
+    "@azure-tools/typespec-azure-resource-manager": ">=0.47.0 <1.0.0",
+    "@azure-tools/typespec-autorest": ">=0.47.0 <1.0.0",
+    "@azure-tools/typespec-client-generator-core": ">=0.47.1 <1.0.0",
+    "@azure-tools/typespec-azure-rulesets": ">=0.47.0 <3.0.0",
+    "@typespec/compiler": ">=0.61.0 <1.0.0",
+    "@typespec/http": ">=0.61.0 <1.0.0",
+    "@typespec/rest": ">=0.61.0 <1.0.0",
+    "@typespec/versioning": ">=0.61.0 <1.0.0",
+    "@typespec/openapi": ">=0.61.0 <1.0.0"
   },
   "dependencies": {
     "js-yaml": "~4.1.0",
     "semver": "~7.6.2"
   },
   "devDependencies": {
-    "@azure-tools/typespec-azure-resource-manager": "~0.46.0",
-    "@azure-tools/typespec-autorest": "~0.46.0",
-    "@azure-tools/cadl-ranch-expect": "0.15.4",
-    "@azure-tools/cadl-ranch-specs": "0.37.2",
+    "@azure-tools/typespec-azure-resource-manager": "~0.47.0",
+    "@azure-tools/typespec-autorest": "~0.47.0",
+    "@azure-tools/cadl-ranch-expect": "0.15.5",
+    "@azure-tools/cadl-ranch-specs": "0.37.5",
     "@types/js-yaml": "~4.0.5",
     "@types/node": "~22.5.4",
     "@types/semver": "7.5.8",
-    "@typespec/openapi": "~0.60.0",
+    "@typespec/openapi": "~0.61.0",
     "c8": "^10.1.2",
     "rimraf": "~6.0.1",
     "typescript": "~5.6.2",
     "typescript-eslint": "^8.5.0",
-    "@azure-tools/typespec-azure-core": "~0.46.0",
-    "@azure-tools/typespec-client-generator-core": "0.46.2",
-    "@typespec/compiler": "~0.60.0",
-    "@typespec/http": "~0.60.0",
-    "@typespec/rest": "~0.60.0",
-    "@typespec/versioning": "~0.60.0",
-    "@azure-tools/typespec-azure-rulesets": "0.46.0",
+    "@azure-tools/typespec-azure-core": "~0.47.0",
+    "@azure-tools/typespec-client-generator-core": "0.47.1",
```

```suggestion
    "@azure-tools/typespec-client-generator-core": "~0.47.1",
```
Shall we add `~` like others?